src/renderer/src/components/settings-modal/settings-modal.tsx

```diff
@@ -24,6 +24,7 @@ const SettingsModal: React.FC<SettingsModalProps> = ({ closeModal }) => {
   const [audioApis, setAudioApis] = useState(Array<AudioApi>);
   const [audioOutputDevices, setAudioOutputDevices] = useState(Array<AudioDevice>);
   const [audioInputDevices, setAudioInputDevices] = useState(Array<AudioDevice>);
+  const [radioEffects, setRadioEffects] = useState("on");
```

Comment:
This sets the default state to `on`, however the native side doesn't have any code that handles `on` as a value.